drivers/hmis/app/graphql/mutations/refresh_external_submissions.rb

```diff
@@ -0,0 +1,20 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
+  class RefreshExternalSubmissions < CleanBaseMutation
+    field :success, Boolean, null: false
+
+    def resolve
+      handlers = ['HmisExternalApis::ConsumeExternalFormSubmissionsJob']
+      return { success: true } if Delayed::Job.queued?(handlers) || Delayed::Job.running?(handlers)
+
+      queue = ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
+      HmisExternalApis::ConsumeExternalFormSubmissionsJob.delay(priority: 0, queue: queue)
```

Comment:
Still to do: Modify the s3 client with the behavior we agreed on (https://github.com/open-path/Green-River/issues/6621#issuecomment-2368996651). I'd like to do this in a separate PR if that's ok.

Reply:
Edit: This is split out into a separate ticket https://github.com/open-path/Green-River/issues/6727